package.json

```diff
@@ -6,9 +6,13 @@
     "not op_mini all"
   ],
   "dependencies": {
+    "i18next": "^19.0.3",
+    "i18next-browser-languagedetector": "^4.0.1",
+    "i18next-xhr-backend": "^3.2.2",
     "nanoid": "2.1.9",
     "react": "16.12.0",
     "react-dom": "16.12.0",
+    "react-i18next": "^11.3.1",
```

Comment:
pin versions

Reply:
👍

Reply:
and re run npm install

Reply:
Done, also delete my node_modules folder and then re-run npm install.

Reply:
oh I see, my PR does not refresh yet give me a minute